Dockerfile

```
@@ -191,6 +191,26 @@ FROM runtime-base AS production
+# EN: Remove pip from the production stage only so shared runtime/dev topology stays intact.
+RUN /opt/venv/bin/python -m pip uninstall -y pip \
+    && /usr/local/bin/python -m pip uninstall -y pip \
+    && /opt/venv/bin/python - <<'PY'
+import importlib.util
+import sys
+
+if importlib.util.find_spec("pip") is not None:
+    sys.stderr.write("pip leaked into production venv\n")
+    sys.exit(1)
+PY
+RUN /usr/local/bin/python - <<'PY'
```

Hadolint (2.14.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)